tests/config/test_is_reserved_keyword.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from briefcase.config import is_reserved_keyword
+
+
+@pytest.mark.parametrize(
+    'name',
+    [
+        'BLACK',
+        'purple',
+        '9pink',
+        'Yellow',
+        'green',
+    ]
+)
+def test_is_not_reserved_keyword_violation(name):
+    "Test that names not in the keywords list return false."
```

Comment:
As above, having comments next to the test cases can be helpful to explain what exactly is being tested (and why). This is especially useful in this case, because a name like `hello world` is *not* a valid name - but it's acceptable here because the check is only validating reserved keywords.